### DISCUSSION TITLE
Sanal ortamda yazilimi nasil test edicez > SIMULATORLER?

### DISCUSSION THREAD

**Discussion by karatasy** [karatasy/ulak-avionic#7] [General]
arduino yada diger elektronik malzemeleri birlestirmeden calisip calismadigini gorebilecegimiz uygulamalar var mi varsa neler

**Comment by Musa-Sina-Ertugrul** [comment]
qemu işlemci ve komutlarının simülatörü bunu virt manager ile gui haline getirebiliyorsun dün yaptım güzel çalışıyor ama sanırım ghz vs seçemiyorsun ama bazı işlemci modellerini seçebiliyorun onlar otamatiktir.

**Comment by karatasy** [comment]
bizim unide elektronik devre icin proteus kullaniliyor

yani proteus uzerinden gidersek daha fazla kisiye katilma imkani olusturabiliriz

hocalara da sorabiliriz
